shells/pipes-shell/web/deploy/webpack.config.js

```diff
@@ -13,22 +13,29 @@ const TerserPlugin = require('terser-webpack-plugin');
 //const Visualizer = require('webpack-visualizer-plugin');
 
 module.exports = {
-  mode: 'none',
+  mode: 'production',
+  //mode: 'none',
   //target: 'node',
   //devtool: 'source-map',
+  performance: {
+    hints: false
+  },
   optimization: {
-     minimize: false,
-     //minimize: true,
-     minimizer: [
-       new TerserPlugin({
-         terserOptions: {
-          mangle: false, // Note `mangle.properties` is `false` by default.
+    //minimize: false,
+    minimize: true,
+    minimizer: [
+      new TerserPlugin({
+        parallel: true,
+        extractComments: false,
+        terserOptions: {
+          //mangle: false, // Note `mangle.properties` is `false` by default.
```

Comment:
Is it intentional that it's commented out?

Reply:
Yes. I left some comments in to illuminate the possibilities, and because sometimes name-mangling can create problems. I thought it useful to have the switches there to flip. But if you prefer a clean config, I can remove that stuff.